client/stylesheets/_toCat.scss

```diff
@@ -1838,7 +1839,7 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
 
 .searchfield {
     text-align: center;
-    margin: -20px 0 10px 0;
+    margin: -20px auto 10px auto !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `-20px auto 10px` instead of `-20px auto 10px auto`